### PR TITLE
Generated client

### PR DESCRIPTION
**Pull request by asmith236**
As part of the field software engineering team effort to standardize Web API client SDKs, a new client SDK has been developed (https://github.com/Keyfactor/keyfactor-go-client-sdk) in Go with more robust support. This pull request ports applications from the old client library to the new SDK to allow clients of the existing library to work with the new SDK without modification.

### CODE REVIEW

**Comment by spbsoluble** [comment]
@asmith236 did you happen to run the tests? Can you include a screenshot or dump of the results? I need to fix the workflow to run the tests but that seems non-trivial at the moment given my nearly non-existent actions skills.

File: api/certificate.go [hunk 1]
```diff
@@ -285,158 +304,125 @@ func (c *Client) DeployPFXCertificate(args *DeployPFXArgs) (*DeployPFXResp, erro
...
-		Query:    &query,
-		Payload:  nil,
-	}
+	xKeyfactorRequestedWith := "APIClient"
+	xKeyfactorApiVersion := "1"
+
+	configuration := keyfactor_command_client_api.NewConfiguration()
+	apiClient := keyfactor_command_client_api.NewAPIClient(configuration)
+
+	resp, _, err := apiClient.CertificateApi.CertificateQueryCertificates(context.Background()).XKeyfactorRequestedWith(xKeyfactorRequestedWith).CollectionId(newQuery.collectionId).IncludeLocations(true).IncludeMetadata(newQuery.includeMetadata).IncludeHasPrivateKey(newQuery.includeHasPrivateKey).Verbose(newQuery.verbose).XKeyfactorApiVersion(xKeyfactorApiVersion).PqQueryString(newQuery.pqQueryString).PqPageReturned(newQuery.pqPageReturned).PqReturnLimit(newQuery.pqReturnLimit).PqSortField(newQuery.pqSortField).PqSortAscending(newQuery.pqSortAscending).PqIncludeRevoked(newQuery.pqIncludeRevoked).PqIncludeExpired(newQuery.pqIncludeExpired).Execute()
```
Comment (spbsoluble) [inline api/certificate.go:409]: Can this be expressed a little cleaner?
```
apiClient.CertificateApi.CertificateQueryCertificates(context.Background())\
  .XKeyfactorRequestedWith(xKeyfactorRequestedWith)
```
This is just hard for a human to read.

File: api/store.go [hunk 2]
```diff
@@ -1,9 +1,11 @@
 package api
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
+	keyfactor_command_client_api "github.com/Keyfactor/keyfactor-go-client-sdk"
```
Comment (spbsoluble) [inline api/store.go:8]: Why is this imported this way and the other files are not?

**Review by spbsoluble — APPROVED** [review]
Mostly looks good to me, just minor formatting issues.

**Review by spbsoluble — APPROVED** [review]

File: api/store.go (hunk 2 above)
Comment (asmith236) [reply api/store.go:8]: I really don't know. The 'keyfactor_command_client_api' portion of the import statement was inserted automatically by Goland. Should I see if I can remove that phrase so the statement matches the other imports?

**Pull request merged**